docs/development_environment.md

```diff
@@ -74,7 +74,7 @@ Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Windows-Subsystem-L
 
 From Windows Store install Ubuntu.
 
-#### Additional steps for Ubuntu 18.04
+#### For those already running Ubuntu 18.04 on WSL
 
 If you chose Ubuntu 18.04 from the Windows Store some additional steps are needed to install the latest FFmpeg version.
```

Comment:
Very small tweak otherwise I think it looks great.
```suggestion
If are running Ubuntu 18.04 on Windows Subsystem for Linux some additional steps are needed to install the latest FFmpeg version.
```

Reply:
Implemented the changes with the missing "you"